shared/featureconfig/flags.go

```diff
@@ -18,6 +18,11 @@ var (
 		Name:  "enable-attestation-cache",
 		Usage: "Enable unsafe cache mechanism. See https://github.com/prysmaticlabs/prysm/issues/3106",
 	}
+	// EnableSSZCache --
+	EnableSSZCache = cli.BoolFlag{
```

Comment:
Can you un-export this and change the name to enableSSZCacheFlag? No need to export it, that's what the config is for.

Reply:
Will do